internal/compiler/parser.rs

```diff
@@ -422,8 +422,9 @@ declare_syntax! {
         /// `[ type ]`
         ArrayType -> [ Type ],
         /// `struct Foo := { ... }
-        StructDeclaration -> [DeclaredIdentifier, ObjectType],
-
+        StructDeclaration -> [DeclaredIdentifier, ObjectType, AtRustAttr],
```

Comment:
i believe it should be

```suggestion
        StructDeclaration -> [DeclaredIdentifier, ObjectType, ?AtRustAttr],
```

To specify that this is optional